CONTRIBUTING.md

```diff
@@ -0,0 +1,62 @@
+# Contributing
+
+To make contributions to this charm, you'll need a working [development setup](https://juju.is/docs/sdk/dev-setup).
+
+You can create an environment for development with `tox`:
+
+```shell
+tox devenv -e integration
+source venv/bin/activate
+```
+
+## Testing
+
+This project uses `tox` for managing test environments. There are some pre-configured environments
+that can be used for linting and formatting code when you're preparing contributions to the charm:
+
+```shell
+tox -e fmt           # update your code according to linting rules
+tox -e lint          # code style
+tox -e static        # static type checking
+tox -e unit          # unit tests
+tox -e integration   # integration tests
+tox                  # runs 'fmt', 'lint', 'static', and 'unit' environments
+```
+
+## Build the charm
+
+Build the charm in this git repository using:
+
+```shell
+charmcraft pack
+```
+
+## Python versions
+
+The charm requires Python 3.12 to run/pack. If your system is using a different version of Python
```

Comment:
> Does `charmcraft` require py3.12 to pack a charm?

You mean in the sense that since the LXD instance that will be used for building the charm will contain Python 3.12, so no need to have this Python version on your local machine? Sure I can omit the pack part.

> Also, what about running? AFAIK your environment's Python version is irrelevant for running the charm, isn't?

I meant running the unit and integration tests here, let me clarify the text

Reply:
WDYT?
5b459fa